parsec_android/android/CMakeLists.txt

```
@@ -125,7 +125,7 @@ message(STATUS "Using install prefix: ${CMAKE_INSTALL_PREFIX}")
 
 add_library( parsec-lib
              SHARED
-             src/main/ext/parsec-lib.cpp )
+             src/main/kotlin/com/oxeanbits/parsec_android/parsec-lib.cpp )
```

is better to put this file somewhere else. 

As suggestion:

```suggestion
             src/main/native/parsec-lib.cpp )
```